internal/cmd/db_list.go

```diff
@@ -23,34 +23,80 @@ var listCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+
 		databases, err := client.Databases.List()
 		if err != nil {
 			return err
 		}
 		setDatabasesCache(databases)
-		var data [][]string
-		var helps []string
-		for _, database := range databases {
-			row := []string{database.Name, getDatabaseLocations(database), getDatabaseUrl(&database)}
-			if len(database.Regions) == 0 {
-				help := fmt.Sprintf("🛠 Run %s to finish your database creation!", internal.Emph("turso db replicate "+database.Name))
-				helps = append(helps, help)
-			}
-			data = append(data, row)
-		}
 
-		sort.Slice(data, func(i, j int) bool {
-			return data[i][0] > data[j][0]
-		})
+		printDBListTable(databases)
+		return nil
+	},
+}
 
-		printTable([]string{"Name", "Locations", "URL"}, data)
+func printDBListTable(databases []turso.Database) {
+	headers, data := dbListTable(databases)
+	if !shouldPrintLocations(databases) {
+		headers, data = removeColumn(headers, data, "Locations")
+	}
+	if !shouldPrintGroups(databases) {
+		headers, data = removeColumn(headers, data, "Group")
+	}
+	printTable(headers, data)
+}
 
-		if len(helps) > 0 {
-			fmt.Println()
-			for _, help := range helps {
-				fmt.Println(help)
-			}
+func shouldPrintLocations(databases []turso.Database) bool {
+	for _, database := range databases {
+		if database.Group == "" {
+			return true
 		}
-		return nil
-	},
+	}
+	return false
+}
+
+func shouldPrintGroups(databases []turso.Database) bool {
+	mp := map[string]bool{}
+	for _, database := range databases {
+		mp[database.Group] = true
```

Comment:
```suggestion
	mp := map[string]struct{}{}
	for _, database := range databases {
		mp[database.Group] = struct{}
```

⛏️: super nit

Reply:
I'll keep this one as it is, as the gains are not relevant in this case